api/internal/owner/serializers.py

```diff
@@ -154,8 +147,18 @@ def validate(self, plan: Dict[str, Any]) -> Dict[str, Any]:
                 detail="You cannot update your plan manually, for help or changes to plan, connect with sales@codecov.io"
             )
 
+        active_plans = list(
+            Plan.objects.select_related("tier").filter(paid_plan=True, is_active=True)
+        )
+        active_plan_names = {plan.name for plan in active_plans}
```

Comment:
```suggestion
        active_plan_names = set(active_plans.values_list("name", flat=True))
```
written from memory so there may be typos, but instead of iterating through, just return the values you want

Reply:
It's cool that django can keep those sets in memory and do additional filters if needed